Code/Framework/AtomCore/AtomCore/Instance/InstanceId.h

```diff
@@ -25,6 +25,13 @@ namespace AZ
         {
             AZ_TYPE_INFO(InstanceId, "{0E59A635-07E8-419F-A0F2-90E0CE9C0AD6}");
 
+            /**
+             * Creates an instance id from an asset. The instance id will share the same guid,
+             * sub id, and an opaque pointer value to a specific version of an asset. This is
+             * an explicit create method rather than a constructor in order to make it explicit.
```

Comment:
nit: saying explicit twice sounds a bit weird. maybe remove the first "explicit"
> This is a create method rather than a constructor in order to make it explicit